mflike/mflike.py

```diff
@@ -33,7 +33,7 @@
 
 If left ``null``, no systematic template is applied.
 
-The values of the systematic parameters are set in ``MFLike.yaml``.  They have to be named as
+The values of the systematic parameters are set in the ``TTTEEE/TTTE/TT/EE/TE/etc.yaml`` fils corresponding to the classes that inherit the ``_MFLike`` one.  They have to be named as
```

Comment:
There is a typo fils -> files. Now that I'm reading this, I'm wondering if the `bandpass_shift` parameters are not best located within `mflike_common.yaml` file since someone reading the `TTTEEE.yaml` file will not see the bandpass shift parameters (considering them as systematics parameters).

Reply:
Thanks for the typo.
Yes I think it's fine too, so to have them all together.